Yoast/wordpress-seo
# Get data for site name, author name and alt description.

## Context


* For the socialPreviews we needed the data for the sitename, authorname and the alt description. This PR makes them available in the redux-store and in the social containers.

## Summary


This PR can be summarized in the following changelog entry:

* Makes sitename, authorname and the alt description available in the containers and the redux-store.

## Relevant technical choices:

* I gathered the data for sitename and authorname in `class-metabox-formatter.php`
* On the JS side the data is gathered at `edit.js` and dispatched to the redux-store.
* With selecters i gather the data in te social containers.
* We need https://github.com/Yoast/javascript/issues/533 to be done to pass this new data to the preview.


## Test instructions

This PR can be tested by following these steps:

* yarn install && composer install && grunt build.
* check in the redux store if under (yoast-seo/editor): `setting.socialpreviews` the `authorName` and the `siteName` are available.
* check if the the `authorName` and the `siteName` and the image.`alt` are in the props of the TwitterWrapper and FacebookWrapper.

## UI changes

## Documentation

## Quality assurance


Fixes #14591
